src/util.cpp

```diff
@@ -226,6 +226,32 @@ ClassicLocale ClassicLocale::instance_;
 
 #endif  // !FLATBUFFERS_LOCALE_INDEPENDENT
 
+std::string BufferToHexText(const void *buffer, size_t buffer_size,  size_t max_length,
```

Comment:
Like I asked last time, if this is used only once, can you inline it where it is called?

Reply:
I have inlined this code but I personally don't think it helps readability/debug-ability.

Reply:
I meant inline `BufferToHexText` itself. It's a function that is only called once. There's no reason for it to be in `util`.

Reply:
Ok, we are on the same page then. That's how is was in the first place before https://github.com/google/flatbuffers/pull/5701/commits/8575c02f2198cd0239e9a95ace7afca037f10d4c. I moved that to the util.cpp based on @vglavnyy suggestion.

Reply:
Ah apologies, didn't see that :)

Reply:
The change you made is still not what I meant though: I meant to inline the function, i.e. make it not a function anymore, and just put its code in the place of the caller. To me, functions that are only called once should not be a function :)

But lets leave it as-is, since @vglavnyy requested it.

Reply:
Ok, thanks for merging! 

Lots if things moving the last few hours. Notice that I move the code out of util.cpp and back into the header the way it originally was. Whatever you want to have done, i'll fix it up. Would be nice if we all feel like the code is up to par.  :)